README.md

```diff
@@ -111,6 +111,20 @@ knativeserving.operator.knative.dev/knative-serving condition met
 sonataflow.sonataflow.org/greeting condition met
 ```
 
+We need to use `initContainers` and `securityContext` in our Knative services, we have to tell Knative to enable that feature:
```

Comment:
maybe worth to document the context: allow SSH key exchange in move2kube workflow

Reply:
ok